### ISSUE TITLE
We should put the updated logos from the style guide here

### ISSUE THREAD

**Issue by timlinux** [qgis/QGIS-Promotion#15]
Also folks are asking for print ready CMYK versions.

**Comment by anitagraser** [comment]
We have CMYK definitions for the colors in http://qgis.org/en/site/getinvolved/styleguide.html but we'll have to check the graphics files to see what definitions are used there ...

**Comment by mbernasocchi** [comment]
@anitagraser is this still open?

**Comment by anitagraser** [comment]
Yes, since https://github.com/qgis/QGIS-Promotion/tree/master/logo only contains SVGs, I guess we don't have any print ready CMYK logos here
